src/tally.cpp

```diff
@@ -59,10 +59,37 @@ pyne::Tally::Tally(std::string type, std::string part_name,
   normalization = norm;
 }
 
-// Destructor
-pyne::Tally::~Tally() {
+pyne::Tally::Tally(std::string part_name, std::string ent_geom,
+                   std::vector<double> orgn,
+                   std::vector<double> mesh_i, std::vector<double> mesh_j, std::vector<double> mesh_k,
+                   std::vector<int> ints_i, std::vector<int> ints_j, std::vector<int> ints_k,
+                   std::vector<double> e_bounds_, std::vector<int> e_ints_,
+                   std::vector<double> axs_, std::vector<double> vec_,
+                   std::string tal_name, double norm) {
+  // Empty Tally Constructor
+  entity_type = "Mesh";
+  entity_name = "";
+  particle_name = pyne::particle::name(part_name);
+  entity_geometry = ent_geom;
+  tally_name = tal_name;
+  entity_size = -1;
+
+  origin = orgn;
+  vec = vec_;
+  axs = axs_;
+  meshes[0] = mesh_i;
+  meshes[1] = mesh_j;
+  meshes[2] = mesh_k;
+  ints[0] = ints_i;
+  ints[1] = ints_j;
+  ints[2] = ints_k;
+  e_bounds = e_bounds_;       ///< Energy Mesh
+  e_ints = e_ints_;  ///< Bin per energy
```

Comment:
Consistent whitespace here.

Reply:
removed the comment there, there were the only one in the constructor, and they should be commented in the header.